testing/shelltests.py

```diff
@@ -0,0 +1,239 @@
+#!/usr/bin/env python3
+import os
+import subprocess
+
+# Configuration
+sqlite_exec = "./target/debug/limbo"
```

Comment:
Let's make this a command line argument so that we can test for compatibility between `sqlite3` and `limbo`?